src/SDKs/_metadata/cognitiveservices_data-plane_CustomSearch.txt

```diff
@@ -1,11 +1,11 @@
-2018-02-08 09:29:49 UTC
+2018-09-13 10:20:32 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      d39e79a6610a5be360b729fcb267aa9d56dafb61
+Commit:      8b107a79bf797084cbbb643b8c65143229bc6783
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\a-ahosma\AppData\Roaming\npm `-- autorest@2.0.4245  
```

Comment:
Please run `msbuild build.proj` from repo root and regenerate the code

Reply:
Build started 9/16/2018 12:10:02 PM.
Project "C:\Users\ahosma\Workspace\azure-sdk-for-net\build.proj" on node 1 (default targets).
C:\Users\ahosma\Workspace\azure-sdk-for-net\tools\bootstrapTools\bootstrap.targets(5,4): error MSB4062: The "GetBuildTo
ols" task could not be loaded from the assembly C:\Users\ahosma\Workspace\azure-sdk-for-net\tools\bootstrapTools\taskBi
naries\Microsoft.Azure.Build.BootstrapTasks.dll. Could not load file or assembly 'Microsoft.Azure.Build.BootstrapTasks,
 Version=0.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. Strong name validation
failed. (Exception from HRESULT: 0x8013141A) Confirm that the <UsingTask> declaration is correct, that the assembly and
 all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framewor
k.ITask. [C:\Users\ahosma\Workspace\azure-sdk-for-net\build.proj]
Done Building Project "C:\Users\ahosma\Workspace\azure-sdk-for-net\build.proj" (default targets) -- FAILED.

I get the error above, I don't remember running msbuild, I always just generated the code using Autorest and that was enough. Why do I need to do this step?

Reply:
Please disable strong name verification. (Sn -Vr) and try again

Reply:
@dsgouda I am getting a different error below
"C:\Users\ahosma\Workspace\azure-sdk-for-net\build.proj" (default target) (1) ->
(GetLatestBuildTools target) ->
  C:\Users\ahosma\Workspace\azure-sdk-for-net\tools\bootstrapTools\bootstrap.targets(5,4)
: error MSB4002: There was a failure retrieving the attributes for parameters in the "Get
BuildTools" task. Could not load type 'Microsoft.Build.Framework.IBuildEngine5' from asse
mbly 'Microsoft.Build.Framework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7
f11d50a3a'. [C:\Users\ahosma\Workspace\azure-sdk-for-net\build.proj]
  C:\Users\ahosma\Workspace\azure-sdk-for-net\tools\bootstrapTools\bootstrap.targets(5,4)
: error MSB4060: The "GetBuildTools" task has been declared or used incorrectly, or faile
d during construction. Check the spelling of the task name and the assembly name. [C:\Use
rs\ahosma\Workspace\azure-sdk-for-net\build.proj]

as I said before, why do I need to do this step? I've made several PRs before and I am sure I did not do msbuild

Reply:
@q3blend please ping offline and I shall resolve the issue